src/resmom/mom_main.c

```diff
@@ -8673,6 +8688,12 @@ main(int argc, char *argv[])
 		(void)fclose(stdin);
 		(void)fclose(stdout);
 		(void)fclose(stderr);
+		dummyfile = fopen("NUL:", "r");
+		assert((dummyfile != 0) && (fileno(dummyfile) == 0));
+		dummyfile = fopen("NUL:", "w");
+		assert((dummyfile != 0) && (fileno(dummyfile) == 1));
+		dummyfile = fopen("NUL:", "w");
+		assert((dummyfile != 0) && (fileno(dummyfile) == 2));
```

Comment:
This is the Windows equivalent to opening "/dev/null". I'm fine with leaving this  "NUL:' file under PBS_HOME/mom_priv. It will get reopened every time windows mom comes up, and serving as its dummy stdin/stdout/stderr, which it doesn't really interact with when running as a service.